### ISSUE TITLE
Continues playing after closing app

### ISSUE THREAD

**Issue by topialla** [cliambrown/EasyNoise#15]
When the app plays and I close the app (on Fairphone 3) by swiping it upwards from the app overview (general square button on bottom right of general android menu) the app continues to play.
I would expect it to stop playing.

Currently I have to open the app again and then press the pause button.

Note I like that it continues to play while I switch to a different app, but it should stop playing when I explicitly close the app (as I can close it with the × from the notifications).

I would give it bug and enhancement since it is a relatively unexpected behaviour.

**Comment by cliambrown** [comment]
@topialla Thanks! But if I understand what you're describing, this is the expected behaviour and is the same as most other audio-playing apps (e.g. podcasts or music) — the noise will keep playing in the background even if the app isn't open, and the only way to stop it is by hitting pause or X in the notification or in the app.

I'll close this for now, but let me know if I've misunderstood the issue.

**Comment by topialla** [comment]
Yes, but the X is not stopping the playing. It only  removes the notification but the app keeps playing. Sometimes the X closes the app but it still keeps playing. So to stop then, I have to start the app and push pause.

There's a difference between open in the background and closing the app. IfI press the X , the app is not open in the background.

**Comment by cliambrown** [comment]
@topialla Ah, yes then that is indeed a bug. Thanks for the info. Would you be able to tell me which version of Android you are on?

**Comment by topialla** [comment]
> @topialla Ah, yes then that is indeed a bug. Thanks for the info. Would you be able to tell me which version of Android you are on?

I'm on 13 but wanted to update today. I'm using a Fairphone 3.
